### PR TITLE
SKW-28: [Web][Skyeex] Routing issues and config handling on detail pages.

### PR DESCRIPTION
**Pull request by CiganOliviu**

### CODE REVIEW

File: src/pages/[slugDetailPage].tsx [hunk 1]
```diff
@@ -8,11 +7,13 @@ import { ServicesParagraph } from "@components/components/Services/Services.css"
...
 import { useGetScreenSize } from "@components/utils/useGetScreenSize";
 import { ProjectsConfig } from "@components/configs/projectsConfig";
-import { getStandardHeader } from "@components/utils/general";
+import Custom404 from "@components/pages/404";
+import { getStandardHeaderForPages } from "@components/pages/index";
 
 const fetchProjectConfigObject = (keyId: string): ProjectsConfigType | undefined => {
-    return ProjectsConfig.find((obj) => obj.id === keyId);
-}
+    // return ProjectsConfig.find((obj) => obj.id === keyId);
```
Comment (CiganOliviu) [inline src/pages/[slugDetailPage].tsx:14]: By commenting this we avoid fetching the project on detail page, thus it results in a 404 page where we check if it is a project accesed and if it is, we redirect to /projects page.

File: src/utils/general.tsx [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-
-export const getStandardHeader = (title: string) => {
-    return (
-        <Head>
-            <title>{title}</title>
-        </Head>
-    )
-};
-
-export const getTextSizeBasedOnBigBreakpoint = (text: string, isBigScreen: boolean): string => {
```
Comment (VassRichard) [inline src/utils/general.tsx:17]: This got removed and never put back. At least at me, I have a compilation error

Comment (CiganOliviu) [reply src/utils/general.tsx:17]: Yes, because of a conflict I solved, sorry. I will solve it now.

**Comment by VassRichard** [comment]
Looks ok, we can merge it after taking out that comment

**Pull request merged**